UnitTests.Silverlight/Moq.Silverlight.Tests.csproj

```diff
@@ -239,4 +242,11 @@
       </FlavorProperties>
     </VisualStudio>
   </ProjectExtensions>
+  <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
```

Comment:
I'm fine moving to Castle nuget, but let us not go down the [old nuget restore rabbit whole](http://www.cazzulino.com/ultimate-cross-platform-nuget-restore.html) ;)